src/ServiceProviders/RepositoryProviders/Bitbucket/BitbucketProvider.php

```diff
@@ -207,6 +207,40 @@ public function commentOnCommit($target_project, $commit_hash, $message)
         ]);
     }
 
+    /**
+     * @inheritdoc
+     */
+    function branchesForPullRequests($target_project, $state)
+    {
+        if (!in_array($state, ['open', 'closed', 'all']))
```

Comment:
This would be more efficiently expressed as an associative array mapping from state to parameters. Optional.

Reply:
Made a change to simplify this code and get rid of the `switch` statement. Let me know if this is what you had in mind.